build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+plugins {
+    id("com.android.application") version "7.1.0" apply false
+    id("com.android.library") version "7.1.0" apply false
+    id("org.jetbrains.kotlin.android") version "1.6.10" apply false
+}
+
+tasks.register("clean", Delete::class) {
+    delete(rootProject.buildDir)
+}
```

Comment:
Diff sometimes "Deleted-Added" rather than "Renamed-Changed". Not sure what the reason for this discrepancy is. :(